diff --git a/node_modules/@sveltejs/kit/src/exports/vite/index.js b/node_modules/@sveltejs/kit/src/exports/vite/index.js
index 7d79bd8..5bf0d9f 100644
--- a/node_modules/@sveltejs/kit/src/exports/vite/index.js
+++ b/node_modules/@sveltejs/kit/src/exports/vite/index.js
@@ -29,6 +29,7 @@ import analyse from '../../core/postbuild/analyse.js';
 import { s } from '../../utils/misc.js';
 import { hash } from '../../utils/hash.js';
 import { dedent, isSvelte5Plus } from '../../core/sync/utils.js';
+import MagicString from 'magic-string';
 import {
 	env_dynamic_private,
 	env_dynamic_public,
@@ -724,7 +725,8 @@ async function kit({ svelte_config }) {
 				// triggering a dev server error. By adding a microtask we ensure that the module is fully loaded
 
 				// Extra newlines to prevent syntax errors around missing semicolons or comments
-				code +=
+				const ms = new MagicString(code);
+				ms.append(
 					'\n\n' +
 					dedent`
 					import * as $$_self_$$ from './${path.basename(id)}';
@@ -738,7 +740,8 @@ async function kit({ svelte_config }) {
 						fn.__.id = ${s(remote.hash)} + '/' + name;
 						fn.__.name = name;
 					}
-				`;
+				`
+				);
 
 				// Emit a dedicated entry chunk for this remote in SSR builds (prod only)
 				if (!dev_server) {
@@ -753,7 +756,7 @@ async function kit({ svelte_config }) {
 					}
 				}
 
-				return code;
+				return { code: ms.toString(), map: ms.generateMap({ hires: true }) };
 			}
 
 			// For the client, read the exports and create a new module that only contains fetch functions with the correct metadata
@@ -801,7 +804,8 @@ async function kit({ svelte_config }) {
 			}
 
 			return {
-				code: result
+				code: result,
+				map: { mappings: '' }
 			};
 		}
 	};
